src/ripple/protocol/TER.h

```diff
@@ -641,6 +642,15 @@ isTecClaim(TER x)
     return ((x) >= tecCLAIM);
 }
 
+namespace detail {
+
```

Comment:
Since other modules are accessing this, it shouldn't be in a `detail` namespace.

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/d558b665cd2658b8fd6d2176662758f9efd8ae3f